common/AppManifest.json

```diff
@@ -3,11 +3,6 @@
         "dependencies": {
             "python": {
                 "version": "3.10"
```

Comment:
Can we completely get rid of this AppManifest.json? its just used inside the install-python.sh script. Maybe lets stick to the approach of pinning the version inside that script like we do for the velocitas cli and kanto.
This AppManifest is kind of confusing because it follows a pretty old approach of the AppManifest and doesn't have any additional use

Reply:
done